set_alias.py

```diff
@@ -1,34 +1,46 @@
 #!/usr/bin/env python
 # Adapted from https://gist.github.com/devdazed/473ab227c323fb01838f
+from __future__ import print_function
+import boto3
+import dateutil.tz
 import json
 import logging
-from datetime import datetime
-from urllib.request import Request, urlopen
-from urllib.error import URLError, HTTPError
-import dateutil.tz
-
-import boto3
+import math
+import requests
 
+from botocore.exceptions import ClientError
+from datetime import datetime, timedelta
+from google.oauth2 import service_account
+from googleapiclient import discovery
 
 import settings
 
 # todo set this up with serverless
 log = logging.getLogger(__name__)
 
+KMS = boto3.client('kms')
+SSM = boto3.client('ssm')
```

Comment:
todo, need to add permissions for ssm need to add permissions to use ssm to lambda role. don't know if there is a way to define the role in the serverless.yml, if not add to setup doc.

Reply:
oh yeah... i forgot i had added it to my lambda_basic_execution role

Reply:
actually looks like we don't need it at all now that i've removed the weekend index param